plaid/api/institutions.py

```diff
@@ -7,7 +7,7 @@ class Institutions(API):
     (`HTTP docs <https://plaid.com/docs/api/#institutions>`__)
     '''
 
-    def get(self, count, offset=0):
+    def get(self, count, offset=0, _options=None):
         '''
         Fetch all Plaid institutions, using /institutions/all.
 
```

Comment:
It doesn't look like `_options` is used anywhere. Please confirm that this is on purpose.

Reply:
it's used in this tests `test_get_by_id_with_include_institution_data`

Reply:
Sorry, I mean it's not passed anywhere — should this not be part of the request?

Reply:
oops will fix